modules/apps/commerce/commerce-api/src/main/resources/com/liferay/commerce/util/packageinfo

```diff
@@ -1 +1 @@
-version 4.1.0
\ No newline at end of file
+version 5.0.0
```

Comment:
Why is a major change?

Reply:
I was waiting for the review of https://github.com/liferay-commerce/liferay-portal/pull/400
So this one has not been rebased yet.

Reply:
btw I guess it is because of the methods added to CommerceShippingHelper.java

Reply:
adding @ProviderType will get this fixed